js/deno.ts

```diff
@@ -6,10 +6,12 @@ export { chdir, cwd } from "./dir";
 export {
   File,
   open,
+  openSync,
   stdin,
   stdout,
   stderr,
   read,
+  readSync,
```

Comment:
You can export all these other sync functions too. Seems useful for people?
Also what about writeSync?